### PR TITLE
ccoctl create-iam-roles should update policies for existing roles

### PR DESCRIPTION
**Pull request by akhil-rane**
For upgrades we need create-iam-roles to update policy for existing
roles with permissions in the new release. We extract credentials
requests manifests from the new release and feed it to
create-iam-roles command.

x-ref: https://issues.redhat.com/browse/CCO-84

### CODE REVIEW

File: pkg/cmd/provisioning/aws/create-iam-roles.go [hunk 1]
```diff
@@ -208,6 +208,11 @@ func createRole(awsClient aws.Client, name string, credReq *credreqv1.Credential
 					role = roleOutput.Role
 					log.Printf("Role %s created", *role.Arn)
 
+					if err := writeCredReqSecret(credReq, targetDir, *role.Arn); err != nil {
+						log.Printf("error: %v", err)
```
Comment (joelddiaz) [inline pkg/cmd/provisioning/aws/create-iam-roles.go:212]: extra debugging output?

**Review by joelddiaz — COMMENTED** [review]
/lgtm

File: pkg/cmd/provisioning/aws/create-iam-roles.go (hunk 1 above)
Comment (akhil-rane) [reply pkg/cmd/provisioning/aws/create-iam-roles.go]: yup, fixed it

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Pull request merged**